runtime/lua/vim/lsp/util.lua

```diff
@@ -1378,6 +1378,18 @@ function M.stylize_markdown(bufnr, contents, opts)
     end
   end
 
+  -- Handle some common html escape sequences
+  stripped = vim.tbl_map(function(line)
+    line = string.gsub(line, '&gt;', '>')
+    line = string.gsub(line, '&lt;', '<')
+    line = string.gsub(line, '&quot;', '"')
+    line = string.gsub(line, '&apos;', "'")
+    line = string.gsub(line, '&ensp;', ' ')
+    line = string.gsub(line, '&emsp;', ' ')
+    line = string.gsub(line, '&amp;', '&')
```

Comment:
This works of course, but this is a good use case for `string.gsub`'s ability to use a table.

```lua
local escapes = {
    ['&gt;'] = '>',
    ['&lt;'] = '<',
    ['&quot;'] = '"',
    ['&apos;'] = "'",
    ['&ensp;'] = ' ',
    ['&emsp;'] = ' ',
    ['&amp;'] = '&',
}

string.gsub(text, '&[^ ;]+;', escapes)
```

Reply:
uuh, nice one. I didn't know gsub could do that. Thanks for the tip. I've updated the PR to use this